pkg/kubelet/types/pod_update_test.go

```diff
@@ -17,6 +17,7 @@ limitations under the License.
 package types
 
 import (
+	"k8s.io/kubernetes/pkg/apis/scheduling"
```

Comment:
move this import down to standard library？seems gofmt will regroup the import sentence.